main.go

```diff
@@ -150,9 +153,15 @@ func runServer(
 	seed int,
 	doBootstrapping bool,
 	bootnodes []pstore.PeerInfo,
-	rpcAddr string) {
+	rpcAddr string,
+	notifierAddr string) {
 	ctx := context.Background()
-	node, err := makeNode(ctx, listenPort, seed, doBootstrapping, bootnodes)
+	eventNotifier, err := NewRpcEventNotifier(ctx, notifierAddr)
+	if err != nil {
+		// TODO: don't use eventNotifier if it is not available
+		eventNotifier = nil
```

Comment:
> maybe the error does not need to be propagated here

Do you mean I should do `eventNotifier, _ := NewRpcEventNotifier(ctx, notifierAddr)` instead?

On second thought, it I think `eventNotifier` might be required as default, since all collations should be verified, except when we perform testing. In this case, we should `log.Fatalf(...)` if `err != nil`. Does this make sense to you?

Reply:
> Do you mean I should do eventNotifier, _ := NewRpcEventNotifier(ctx, notifierAddr) instead?

I mean `NewRpcEventNotifier` probably does not need to return `error`.

> On second thought, it I think eventNotifier might be required as default

In my opinion there are functionalities other than sending/receiving collation in this package and those functionalities should be able to keep functioning with/without collation verification.

Reply:
> I mean NewRpcEventNotifier probably does not need to return error.

However, in other use cases which require `eventNotifier`, it might still be important to know if `NewRpcEventNotifier` is successfully New'ed?

Reply:
> In my opinion there are functionalities other than sending/receiving collation in this package and those functionalities should be able to keep functioning with/without collation verification.

Alright I agree.

Reply:
> However, in other use cases which require eventNotifier, it might still be important to know if NewRpcEventNotifier is successful?

Yeah makes sense.